project/Dependencies.scala

```diff
@@ -74,13 +76,30 @@ object Dependencies {
       Nil
     }
 
+  val extractVs = "(\\d+)\\.(\\d+)\\.(\\d+).*".r
   def sparkHive(v: String) = "org.apache.spark" %% "spark-hive" % v excludeAll(
     ExclusionRule("org.apache.hadoop"),
     ExclusionRule("org.apache.ivy", "ivy"),
     ExclusionRule("javax.servlet", "servlet-api"),
-    ExclusionRule("org.mortbay.jetty", "servlet-api"),
-    ExclusionRule("com.twitter", "parquet-hadoop-bundle")
-  ) excludeAll(parquetList:_*)
+    ExclusionRule("org.mortbay.jetty", "servlet-api")
+  ) excludeAll(parquetList:_*) excludeAll(
+    {
+      val s = defaultSparkVersion match { case extractVs(v, m, p) =>  (v.toInt, m.toInt, p.toInt)}
+      val h = defaultHadoopVersion match { case extractVs(v, m, p) => (v.toInt, m.toInt, p.toInt)}
+      if(
```

Comment:
how about:
```scala
import scala.math.Ordering.Implicits._
sparkVersion >= (1, 5, 2)
hadoopVersion >= (2.6.0)
```
and this would be simpler and less buggy for future versions.